gateways/adyen/translation.go

```diff
@@ -0,0 +1,49 @@
+package adyen
+
+import "github.com/BoltApp/sleet"
+
+// translateCvv converts a Adyen CVV response code to its equivalent Sleet standard code.
+func translateCvv(rawCvv string) sleet.CVVResponse {
+	// Codes taken from: https://docs.adyen.com/development-resources/test-cards/cvc-cvv-result-testing
+	switch rawCvv {
+	case "0":
+		return sleet.CVVResponseNoResponse
+	case "5":
+		return sleet.CVVResponseUnsupported
+	case "1":
+		return sleet.CVVResponseMatch
+	case "2":
+		return sleet.CVVResponseNoMatch
+	case "3", "6":
+		return sleet.CVVResponseNotProcessed
+	case "4":
+		return sleet.CVVResponseRequiredButMissing
+	default:
+		return sleet.CVVResponseUnknown
+	}
+}
+
+// translateAvs converts a Adyen AVS response code to its equivalent Sleet standard code.
+func translateAvs(rawAvs string) sleet.AVSResponse {
+	// Codes taken from: https://docs.adyen.com/risk-management/avs-checks
+	switch rawAvs {
+	case "A":
```

Comment:
Is there a way to get Adyen's mapped response code instead of the raw code? If not we'll probably need to get the processor from the BIN so we can differentiate duplicate-meaning codes like "A".

Reply:
@jakechiang-bolt this is done now - I strongly typed what I've found from Adyen's codes and playing with postman